modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/view_object_entries.jsp

```diff
@@ -18,17 +18,30 @@
 
 <%
 ViewObjectEntriesDisplayContext viewObjectEntriesDisplayContext = (ViewObjectEntriesDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+ObjectDefinition objectDefinition = viewObjectEntriesDisplayContext.getObjectDefinition();
 %>
 
-<clay:headless-data-set-display
-	apiURL="<%= viewObjectEntriesDisplayContext.getAPIURL() %>"
-	clayDataSetActionDropdownItems="<%= viewObjectEntriesDisplayContext.getClayDataSetActionDropdownItems() %>"
-	creationMenu="<%= viewObjectEntriesDisplayContext.getCreationMenu() %>"
-	formName="fm"
-	id="<%= viewObjectEntriesDisplayContext.getClayHeadlessDataSetDisplayId() %>"
-	itemsPerPage="<%= 20 %>"
-	namespace="<%= liferayPortletResponse.getNamespace() %>"
-	pageNumber="<%= 1 %>"
-	portletURL="<%= liferayPortletResponse.createRenderURL() %>"
-	style="fluid"
-/>
\ No newline at end of file
+<c:choose>
+	<c:when test="<%= objectDefinition.isPortlet() %>">
```

Comment:
Just a question: When the object is inactive, does this isPortlet method return `false`? Because I think we can use `isActive()` instead. What do you think?

Reply:
Hi @gabrielwas, When the object is inactive or active, `isPortlet()` does not change the value. If we use `isActive`, we might have this situation: Object active, but display set to `Hide Widget` and object will continue to be shown on the page, because the `isPortlet()` is false. So I think we have to use the `isPortlet()`

Reply:
But, if we want to bring this alert to inactive functionality, we can add a new condition too, what you think?

Reply:
Sorry for the misunderstanding, Daniel. I thought that this behavior was for the active/inactive process. You're right. Thanks.

Reply:
Hey @danvb112 , unfortunately this approach will not work, because the content is being hidden even when the Portlet is accessed from menu. To use this solution, we would need to know if the Portlet is being rendered inside a Display Page.